.eslintrc.json

```diff
@@ -22,7 +22,8 @@
     ],
     "ignorePatterns": ["**/*.js", "coverage/**"],
     "rules": {
-      "no-console": "warn"
+      "no-console": "warn",
+      "camelcase": "off"
```

Comment:
So the only place we really want to allow non-camel-case is in the models, right? Because those are required to be snake case per the API specification? Thoughts on just exempting `models/**.ts` in the `overrides` section below? Something like this?
```
{
  "files": ["models/**.ts"],
  "rules": { "camelcase": "off" }
}
```

I'd like to avoid letting snake case sneak into the rest of the app code or the infrastructure. And we can exempt single lines as needed with `eslint-disable-line camelcase` later if we need to?

Reply:
I'm going to reverse this and deal with it later.   I was trying to avoid having the linter suggest too many changes related to camelCase.  But that doesn't really belong in this PR with the function impl.

Reply:
Reversed with https://github.com/dod-ccpo/atat-web-api/pull/40/commits/ecf1410cd1c61bb71bffe8dbdba0dadcf228ab9b